### ISSUE TITLE
[FastPitch 1.1/PyTorch] fine tuning a trained multi-speaker model for a new speaker

### ISSUE THREAD

**Issue by dsplog** [NVIDIA/DeepLearningExamples#1027]
Related to **FastPitch 1.1/PyTorch** 

**Describe the bug**
This is not a bug, but trying to understand how to fine tune a trained multi-speaker model for a new speaker. 

**To Reproduce**
1. Train from scratch a multi speaker model with ~40 speakers
2. Now a new speaker dataset is available with ~30minutes of data. 
3. To fine tune the model for the new speaker, tried the following : 
- added the new speaker dataset to existing train/test split data
- re-started the training to load from the last saved epoch
- no parameters like WARMUP_STEPS/KL_LOSS_WARMUP was touched

waited for ~20 epochs after the restart and checked. could see that the model has not learned the new speaker. the existing speakers where working fine. 

Had read through https://github.com/NVIDIA/DeepLearningExamples/issues/1004 , but did not follow it sufficiently. Requesting your help to provide some insights.

**Expected behavior**
Was hoping to get a way to fine tune for a new speaker, but requesting guidance on your thoughts, or any tweaks needed for learning rate etc 

**Environment**
N/A
